crypto/fipsmodule/ec/make_tables.go

```diff
@@ -208,32 +214,27 @@ func writeP384Table(path string) error {
 // P-384 base point pre computation
 // --------------------------------
 //
-// The precomputed table for the base point G of P-384, fiat_p384_g_pre_comp,
-// consists of 14 sub-tables, each holding 64 points. A point is represented
+// The precomputed table for the base point G of P-384, |p384_g_pre_comp|,
```

Comment:
Same suggested beginning of the sentence as for P-521.

Reply:
done